govcd/external_network_v2.go

```diff
@@ -0,0 +1,189 @@
+/*
+ * Copyright 2020 VMware, Inc.  All rights reserved.  Licensed under the Apache v2 License.
+ */
+
+package govcd
+
+import (
+	"fmt"
+	"net/url"
+
+	"github.com/vmware/go-vcloud-director/v2/types/v56"
+)
+
+// ExternalNetworkV2 is a type for version 2 of external network which uses OpenAPI endpoint to manage external networks
+type ExternalNetworkV2 struct {
+	ExternalNetwork *types.ExternalNetworkV2
+	client          *Client
+}
+
+// CreateExternalNetwork creates a new external network using OpenAPI endpoint
```

Comment:
I think it's worth to mentioned similar things like it is in terraform. Like it create nsx-v, nsx-t and which fields supported by which type.

Reply:
Added more details